bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -1149,37 +1149,34 @@
             `property_value`=@concept_set_uuid;
         </sql>
     </changeSet>
-    <changeSet id="bahmni-core-201906101318" author="Vinay, Indraneel, Samuel34">
+    <changeSet id="bahmni-core-202304192116" author="Angshu">
         <preConditions onFail="MARK_RAN">
-        	<sqlCheck expectedResult="1">
-        		select count(*) from global_property where property = 'order.durationUnitsConceptUuid' and property_value IS NULL;
-        	</sqlCheck>
+            <sqlCheck expectedResult="1">
+                select count(*) from global_property where property = 'order.durationUnitsConceptUuid' and property_value IS NULL;
+            </sqlCheck>
         </preConditions>
-        <comment>Adding duration unit concepts and setting up associated global property</comment>
+        <comment>Adding duration unit concepts and setting up associated global property. CIEL dataset should create the actual members. this is replacement of changeset bahmni-core-201906101318</comment>
         <sql>
             set @concept_id = 0;
             set @concept_name_short_id = 0;
             set @concept_name_full_id = 0;
             call add_concept(@concept_id, @concept_name_short_id, @concept_name_full_id, 'Duration Units', 'duration units', 'N/A', 'ConvSet', true);
```

Comment:
As we are changing the changeset id, when `add_concept` executes on an existing DB, wouldn't this cause conflicts or errors ?

Reply:
Precondition would resolve this